docker-compose.yaml

```diff
@@ -153,7 +165,8 @@ services:
       KEYCLOAK_INIT_TASKLIST_ROOT_URL: http://localhost:8082
       KEYCLOAK_INIT_OPTIMIZE_SECRET: XALaRPl5qwTEItdwCMiPS62nVpKs7dL7
       KEYCLOAK_INIT_OPTIMIZE_ROOT_URL: http://localhost:8083
-      KEYCLOAK_INIT_WEBMODELER_ROOT_URL: http://localhost:8070   
+      KEYCLOAK_INIT_WEBMODELER_ROOT_URL: http://localhost:8070  
+      KEYCLOAK_INIT_ZEEBE_NAME: zeebe
```

Comment:
good point, will remove it from the env file 👍

Reply:
@megglos `KEYCLOAK_INIT_ZEEBE_NAME: zeebe` should have stayed, otherwise the Zeebe api will be not be initialized resulting in an error in Identity on startup and when an app tries to authenticate with `zeebe-api` audience.

Reply:
apologies for the confusion, I meant the `ZEEBE_TOKEN_AUDIENCE` env var with the removal, the `KEYCLOAK_INIT_ZEEBE_NAME` stayed

Reply:
@megglos my bad, I checked the files and did not find the var, but it is actually there 🙈 I have probably selected the wrong commit